File: Server_Sokoban/src/main/java/com/mycompany/server_sokoban/comunication/SimpleMessage.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package com.mycompany.server_sokoban.comunication;

/**
 * 
 * @author Estuardo Ramos
 */
import java.io.Serializable;

public class SimpleMessage implements Serializable {

    private String message;

    private static final long serialVersionUID = 6529685098267757690L;

    public SimpleMessage() {
    }

    public SimpleMessage(String message) {
        this.message = message;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    @Override
    public String toString() {
        return "SimpleMessage{" +
                "message='" + message + '\'' +
                '}';
    }
}

File: Game1/app/src/main/java/com/example/game1/comunication/SimpleMessage.kt
package com.example.game1.comunication

class SimpleMessage(val message: String) : java.io.Serializable {

    companion object {
        private const val serialVersionUID = 6529685098267757690L
    }

    override fun toString(): String {
        return "SimpleMessage(message='$message')"
    }
}